docker-compose.yml

```diff
@@ -1,23 +1,21 @@
-version: '3.0'
-services:
+version: '3.8'
 
+services:
   elasticsearch:
     image: amsterdam/elasticsearch6
-    command:  elasticsearch -Ehttp.host=0.0.0.0 -Etransport.host=127.0.0.1
-    volumes:
-      - "~/.ssh/datapunt.key:/root/.ssh/datapunt.key"
```

Comment:
This volume is no longer required? Seems needed to establish a connection to the internal API routes.

Reply:
The volume is an empty folder and doesn't appear to be used. Looks like its a copy-paste action from some time ago, because I came across this volume definition in the `cms` repo as well.